contrib/cirrus/runner.sh

```diff
@@ -29,11 +29,26 @@ function _run_validate-source() {
 
     # make sure PRs have jira links (if needed for branch)
     showrun make test-jira-links-included
+
+    # Defined by Cirrus-CI for all tasks
```

Comment:
All the commits look great, except this one. I realize `validate` is only run in PRs (via `.cirrus.yml`) but I still don't like this here. Could I ask you to add, perhaps on line 25, `req_env_vars CIRRUS_CHANGE_IN_REPO PR_BASE_SHA` and a nice comment like "This target should only run on PRs" ? Not a blocker, just a request from future me.

Reply:
yes that makes sense, it certainly should create a proper error message if it is ever called from the wrong context instead of failing in a undefined way.